rosdiagnostic/package.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0"?>
+<package format="2">
+  <name>rosdiagnostic</name>
+  <version>1.8.10</version>
+  <description>The rosdiagnostic package</description>
+
+  <author email="gautran@clearpath.ai">Guillaume Autran</author>
+  <maintainer email="gautran@clearpath.ai">Guillaume Autran</maintainer>
+  <license>BSD</license>
+
+  <buildtool_depend>catkin</buildtool_depend>
+  <exec_depend>rospy</exec_depend>
```

Comment:
Depend on the messages you need.

Also drop the `<export>` element; not needed.

And fill in a proper description, please.